Assets/Scripts/Player.cs

```diff
@@ -49,9 +51,60 @@ public class Player : MonoBehaviour
 
 	private string openVideo;
 
-	void Start ()
+	private static class ConnectedDevices
 	{
-		XRSettings.enabled = false;
+		public enum LoadedDevice
+		{
+			None,
+			Oculus,
+			OpenVr
+		}
+
+		public static LoadedDevice loadedDevice;
+
+		public static bool hasLeftController { get; set; }
```

Comment:
Any public variable that does not have a backing field, you don't have to add the default getter and setter for.